### PR TITLE
First time setup guide

### PR DESCRIPTION
**Pull request by sigvef**

### CODE REVIEW

File: guides/mbed-org.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+layout: guide
+---
+
+# First-time setup
+
+To start developing for the mbed-enabled NRF51-DK, you first need to register an account on the online compiler at [developer.mbed.org](https://developer.mbed.org) and associate the development board model with your account.
```
Comment (cristeahub) [inline guides/mbed-org.md]: `nRF51-DK`

File: guides/mbed-org.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+This is an easy process.
+
+Visit [developer.mbed.org](https://developer.mbed.org) and click "Login or signup".
+
+![Visit yo](images/mbed-org-1.png)
```
Comment (cristeahub) [inline guides/mbed-org.md]: Concider changing alt text to something more descriptive

File: guides/mbed-org.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+Open the `mbed.htm` file that is on this device.
+
+![Visit yo](images/mbed-org-6.png)
+
+You should automatically be taken to mbed.org with a notification like this one:
+
+![Visit yo](images/mbed-org-7.png)
+
+You have now successfully made an account on mbed.org, and have added associated your development board hardware model with your account.
+This means that you are ready to start developing!
```
Comment (cristeahub) [inline guides/mbed-org.md:41]: Maybe add a like (tentative for now) to the `your first program` guide

Comment (sigvef) [inline guides/mbed-org.md:41]: A like?

File: guides/mbed-org.md (hunk 2 above)
Comment (sigvef) [inline guides/mbed-org.md]: Woops, accidental inclusion there.

File: guides/mbed-org.md (hunk 3 above)
Comment (cristeahub) [inline guides/mbed-org.md:41]: A link\* sorry

File: guides/mbed-org.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+The board will show up on your computer as a regular USB storage device.
+Open the `mbed.htm` file that is on this device.
+
+![Visit yo](images/mbed-org-6.png)
+
+You should automatically be taken to mbed.org with a notification like this one:
+
+![Visit yo](images/mbed-org-7.png)
+
+You have now successfully made an account on mbed.org, and have added associated your development board hardware model with your account.
```
Comment (stianjensen) [inline guides/mbed-org.md]: added associated

File: guides/mbed-org.md (hunk 1 above)
Comment (sigvef) [inline guides/mbed-org.md]: Are we still using the dk btw?

Comment (cristeahub) [inline guides/mbed-org.md]: They've got at least 15 of them for us, hopefully all 30 before the first course. We're picking them up this week

**Comment by sigvef** [comment]
ping

File: guides/mbed-org.md (hunk 1 above)
Comment (stianjensen) [inline guides/mbed-org.md]: Ah, I didn't realize that was the DK. Cool!

**Pull request merged**